.gitignore

```diff
@@ -1 +1,2 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+node_modules/
```

Comment:
Generally, I think the preference is to not commit node_modules since they have a tendency to balloon in size pretty quickly -- granted, I'm not a JS/TS dev professionally, so maybe my understanding is wrong -- and it's quick enough to just run a `<npm | bun | pnpm> install`